model/Software/Properties/homepage.md

```diff
@@ -1,22 +1,22 @@
 SPDX-License-Identifier: Community-Spec-1.0
 
-# homePage
+# homepage
 
 ## Summary
 
 A place for the SPDX document creator to record a website that serves as the package's home page.
 
 ## Description
 
-HomePage is a place for the SPDX document creator to record a website that serves as the package's home page.
+Homepage is a place for the SPDX document creator to record a website that serves as the package's home page.
 This saves the recipient of the SPDX document who is looking for more info from
 having to search for and verify a match between the package and the associated project home page.
 This link can also be used to reference further information about the package
 referenced by the SPDX document creator.
 
 ## Metadata
 
-- name: homePage
+- name: doap:homepage
```

Comment:
we also have no file for `xsd:string` and  the filename is not `daop:hompepage.md`. This is some inconsistency that I can not resolve. I do not see that this is matching the expectations of the spec-parser.


I think that using `doap:homepage` implies removing our file `model/Software/Properties/homepage.md`